### ISSUE TITLE
"Error: @match: Invalid protocol (http*:) specified." when attempting to download the script

### ISSUE THREAD

**Issue by SacredDuckwhale** [Kir-Antipov/GitHub-Defreshed#3]
This was on Firefox 77, using GreaseMonkey, but Chrome also refused to download it from the link provided on the repository's main page (https://raw.githubusercontent.com/Kir-Antipov/GitHub-Defreshed/master/build/github-defreshed.user.js).

The script itself worked after manually building it with npm and replacing the ``http*`` part with ``https`` before importing it.

Thanks for your efforts, by the way! The repository page at least looks much less anger-inducing now, even if it took some custom CSS to undo the horrible style changes.

**Comment by Kir-Antipov** [comment]
Thank you much for your feedback!

> The script itself worked after [...] replacing the `http*` part with `https` 

I'd have to check it myself, but I calmed down at the thought that Tampermonkey uses a unified header format 😸
Well, GitHub automatically redirects users to the https protocol, so replacing it in the script won't be a problem. I'll fix this in few minutes

> if it took some custom CSS to undo the horrible style changes

I focused on fixing the most terrible changes, which are terrible objectively and for everyone. Some found these style changes even pleasant, and I’m just cold to them (perhaps, against the unforgivable layout change, it just doesn’t seem like such a nightmare to me xD)

However, I also have plans for the optional old styles reviving for those who don't like the new ones
